src/Type/Php/StrlenFunctionReturnTypeExtension.php

```diff
@@ -30,8 +35,31 @@ public function getTypeFromFunctionCall(
 		}
 
 		$argType = $scope->getType($args[0]->value);
+
+		$constantStrings = TypeUtils::getConstantStrings($argType);
+		$min = null;
+		$max = null;
+		foreach($constantStrings as $constantString) {
+			$len = strlen($constantString->getValue());
+
+			if ($min === null || $len < $min) {
+				$min = $len;
+			}
+			if ($len > $max) {
+				$max = $len;
+			}
+		}
+		if ($min || $max) {
+			return IntegerRangeType::fromInterval($min, $max);
+		}
+
+		if ($argType instanceof BooleanType) {
```

Comment:
I tried using `BooleanType->isSuperTypeOf()` for this case, but this made the empty-string case fail.
not sure whether using `$argType instanceof BooleanType` is "ok" in this case, since you did not added a comment on this line

Reply:
resolved...typo